platforms/documentation/docs/src/docs/userguide/reference/gradle_daemon.adoc

```diff
@@ -212,40 +215,111 @@ The following file will be generated:
 [source,properties]
 ----
 #This file is generated by updateDaemonJvm
+toolchainUrl.FREE_BSD.AARCH64=https\://example.com/...
+toolchainUrl.FREE_BSD.X86_64=https\://example.com/...
+toolchainUrl.LINUX.AARCH64=https\://example.com/...
+toolchainUrl.LINUX.X86_64=https\://example.com/...
+toolchainUrl.MAC_OS.AARCH64=https\://example.com/...
+toolchainUrl.MAC_OS.X86_64=https\://example.com/...
+toolchainUrl.UNIX.AARCH64=https\://example.com/...
+toolchainUrl.UNIX.X86_64=https\://example.com/...
+toolchainUrl.WINDOWS.X86_64=https\://example.com/...
```

Comment:
Something to consider is mentioning about the possibility of configuring using local paths instead by using  `file\:/<LOCAL_ABSOLUTE_PATH>` similar to the wrapper `distributionUrl`.  Definitely this is quite advance usage and not the expected but it's supported

Reply:
Let's not push this too much in front, as this might be the kind of thing we end up reworking when making toolchains behave more like regular dependencies.

Reply:
Will add at a later time to docs.